tardis/montecarlo/src/integrator.c

```diff
@@ -246,10 +255,13 @@ _formal_integral(
                               &idx_nu_start
                   );
 
+
                   // Initialize pointers for inner loop
                   pline = storage->line_list_nu + idx_nu_start;
                   pexp_tau = exp_tau + offset + idx_nu_start;
                   patt_S_ul = att_S_ul + offset + idx_nu_start;
+                  pJred_lu = Jred_lu + offset + idx_nu_start;
+                  pJblue_lu = Jblue_lu + offset + idx_nu_start;
 
                   for (;pline < storage->line_list_nu + size_line;
```

Comment:
maybe I can change `pline < storage->line_list_nu + size_line` to 
`(pline < storage->line_list_nu + size_line) & (*pline < nu_end)`

Alternatively one could add `idx_nu_end` and then change the loop condition to `pline < storage->line_list_nu + idx_nu_end`.